bootstrap.yml

```diff
@@ -760,6 +757,8 @@ packages:
         - 'meson'
         - '--prefix=/usr'
         - '--libdir=lib'
+        # Install to /usr/share to avoid conflicts with standard C++ headers.
```

Comment:
Going to put it here, but the same comments from cxxshim apply for frigg, while we’re working on it, we can fix it, saves us fixing later.